### ISSUE TITLE
Allow schema specification for SQL databases

### ISSUE THREAD

**Issue by cpcloud** [ContinuumIO/odo#220]
```python
odo(pd.DataFrame([{'a': 1}, {'a': 1}]), 'postgresql://localhost/bz::schema.tmp')
```

or 

```python
odo(pd.DataFrame([{'a': 1}, {'a': 1}]), 'postgresql://localhost/bz.schema::tmp')
```

**Comment by cpcloud** [comment]
cc @llllllllll

**Comment by llllllllll** [comment]
I personally prefer the first one because that would be the syntax used to reference the table in postgres

**Comment by cpcloud** [comment]
cool, let me dig up the old issue and see what that looked like

**Comment by llllllllll** [comment]
If you give me a good starting point, I could work on this. I would really like this feature so I don't mind building it

**Comment by cpcloud** [comment]
https://github.com/ContinuumIO/blaze/pull/803 is the original issue

**Comment by cpcloud** [comment]
looks like we did support it. i suspect it was lost when odo was ripped out of blaze

**Comment by cpcloud** [comment]
closed by #223

**Comment by dvreed77** [comment]
What is the proper way to reference the schema? I've tried both ways referenced above and neither work. I can't determine it from the code changes either. Thanks for the help
